### PR TITLE
Remove stratified inlining

### PR DESCRIPTION
**Pull request by akashlal**
For #200

### CODE REVIEW

File: Source/VCGeneration/Couterexample.cs [hunk 1]
```diff
@@ -182,7 +182,7 @@ public void Print(int indent, TextWriter tw, Action<Block> blockAction = null)
               {
                 var cmd = getTraceCmd(loc);
                 var calleeName = getCalledProcName(cmd);
-                if (calleeName.StartsWith(VC.StratifiedVCGen.recordProcName) &&
+                if (calleeName.StartsWith(VC.StratifiedVCGenBase.recordProcName) &&
```
[the remark this answers is not in the document]
Comment (akashlal) [reply Source/VCGeneration/Couterexample.cs:185]: StratifiedVCGenBase is an abstract class. Cannot be used directly, so perhaps Base is appropriate. I'll add a comment saying that the class is instantiated by Corral.

File: Source/Core/CommandLineOptions.cs [hunk 2]
```diff
@@ -741,11 +741,7 @@ public enum Inlining
     public int StratifiedInlining = 0;
     public string FixedPointEngine = null;
     public bool StratifiedInliningWithoutModels = false; // disable model generation for SI
```
[the remark this answers is not in the document]
Comment (akashlal) [reply Source/Core/CommandLineOptions.cs]: Done

**Review by shazqadeer — APPROVED** [review]
Thanks a lot!

**Comment by shazqadeer** [comment]
Closes #200

**Pull request merged**